packages/suite/src/support/suite/ConditionalRender.tsx

```diff
@@ -0,0 +1,41 @@
+import { useResponsiveContext } from './ResponsiveContext';
+
+export type ContainerType = 'content' | 'sidebar';
+
+export type UseConditionalRenderProps = {
+    container: ContainerType;
+    minWidth?: number;
+    maxWidth?: number;
```

Comment:
I guess it doesn't make sense to pass `width` and `maxWidth` / `minWidth` together?

Like this `maxWidth={x] minWidth={y} width={m}` doesn't make sense, right? It could be written in types which combinations are valid or not.

But that's up to you.

Reply:
Done